pkgs/development/python-modules/google-music-proto/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi
+, attrs
+, audio-metadata
+, marshmallow
+, pendulum
+, protobuf
+}:
+
+buildPythonPackage rec {
+  pname = "google-music-proto";
+  version = "2.1.2";
+
```

Comment:
Needs `disabled` for constrained python interpreter versions https://github.com/thebigmunch/google-music-proto/blob/3f7f81cced963a01bc9dfbbc7cd44ca6092c3aa9/pyproject.toml#L20